test/controllers/articles_controller_test.rb

```diff
@@ -3,28 +3,86 @@
 require "test_helper"
 
 # Controller tests for news articles
-class ArticleControllerTest < FunctionalTestCase
-  def test_create_article_get
+class ArticlesControllerTest < FunctionalTestCase
+  ############ test Actions that Display data (index, show, etc.)
+
+  def test_index
+    # Prove any user can see article index
+    get(:index_articles)
+    assert(:success)
```

Comment:
1. There's a bug here and in line 23.
```suggestion
    assert_response(:success)
```
2. Should have a test for content.
Contrary to the comment in the code, this does not prove that the user sees anything. There's a similar misleading comment in the [RailsGuide](https://guides.rubyonrails.org/testing.html#what-to-include-in-your-functional-tests) about `assert_response(:success)`. ("ensur[es]that the right response body has been generated.")